pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataTeardown.java

```diff
@@ -99,11 +113,22 @@ public static void main(String[] args) throws Exception {
             throw e;
         }
 
+        if (arguments.metadataStoreUrl == null && arguments.zookeeper == null) {
+            System.err.println("Metadata store address argument is required (--metadata-store)");
+            commander.usage(commander.getOut());
+            System.exit(1);
```

Comment:
DOT USE `System.exit(1)`. In the testing environment, it may cause the test to exit.

```java
        if (arguments.metadataStoreUrl == null && arguments.zookeeper == null) {
            commander.usage(commander.getOut());
            throw new IllegalArgumentException("Metadata store address argument is required (--metadata-store)");
        }
```

Reply:
Updated.